runtime/plugins/comment/comment.lua

```diff
@@ -60,6 +60,7 @@ ft["yaml"] = "# %s"
 ft["zig"] = "// %s"
 ft["zscript"] = "// %s"
 ft["zsh"] = "# %s"
+ft["openscad"] = "// %s"
```

Comment:
This should probably go between "ocaml" and "pascal" to maintain the alphabetic ordering.